CONVENTIONS.md

```diff
@@ -19,7 +19,16 @@ We use the functions exported from [libc][libc] instead of writing our own
 `extern` declarations.
 
 We use the `struct` definitions from [libc][libc] internally instead of writing
-our own.
+our own. If we want to add methods to a libc type, we use the newtype pattern.
+For example,
+
+```rust
+pub struct SigSet(libc::sigset_t);
+
+impl SigSet {
```

Comment:
I agree. So, let's add another sentence that mentions that we use the rust convention of camel case names for structs. Then we can merge this PR.